test/CMakeLists.txt

```diff
@@ -92,6 +92,20 @@ if( iodaconv_bufr_ENABLED )
   )
 endif()
 
+if( iodaconv_pbfortran_ENABLED )
+  list( APPEND test_input
+    testinput/gnssro.bufr
+    testinput/prepbufr.bufr
+    testinput/amsua.bufr
+  )
+
+  list( APPEND test_output
+    testoutput/gnssro_obs_2020093018.nc4
```

Comment:
We already have GNSSRO, MHS, etc. BUFR files in this repository, can we reuse them instead of adding new ones? Do we already have a PrepBUFR file too?